### ISSUE TITLE
Create provenance resources in MELD sandbox

### ISSUE THREAD

**Issue by savanahmueller** [chronic-care/mcc-project#272]
When we populate new data in the sandboxes, it will display a Source.
Start with the regular Care Planning sandbox.

**Comment by kbertodatti** [comment]
The data is in the sandbox but it's not displaying. Not sure if this is the case because of the CQL or because of how the data is set up.

Sean to ask @drdavec

**Comment by drdavec** [comment]
@swmuir, which MELD sandbox are you using to test this?  I looked at "Care Planning" to review the Provenance resources, but there are no Provenance resources in this sandbox.

**Comment by swmuir** [comment]
@drdavec 
https://gw.interop.community/MCCDevelopment/data

Provenance?target=Condition/mcc-cond-pnoelle-hypertension
This also pulls it in 
Condition?patient=mcc-pat-pnoelle&_revinclude=Provenance:target
I am getting the fhir blob but the CQL does not seem to find it when processing

**Comment by savanahmueller** [comment]
@swmuir is adding @drdavec to Develop sandbox to review.

**Comment by drdavec** [comment]
The Provenance resource display is now working in MELD CarePlanning sandbox and can be tested there.  Provenance source is displayed for Hypertension and Dyslipidemia conditions.  I found these issues:

1) The example Provenance in MCC Development sandbox used a target reference = "Condition/mcc-cond-pnoelle-hypertension/_history/1".  Remove the version-specific "/history/1" suffix.  The CQL logic cannot resolve this history version reference, and I've never seen any EHR system FHIR API that supports history queries, so we won't see this from Epic.

2) I added two new Provenance resources for hypertension and dyslipidemia using a Provenance resource template that is exactly what we will see from Epic production systems. I recommend using this template for additional samples because it is based on US Core requirements for two agents: author and transmitter.  And CQL is designed to support this agent style.  See attached sample (also in MCC Development and Care Planning sandboxes).

3) The provenance source still was not displayed.  I found and fixed a bug where MyCarePlanner used a query format of "&_revInclude" (with a capital I) instead of "&_revinclude" (with lower-case i).  The upper-case I works for Epic, but is ignored by MELD.  I committed this change to develop branch.

4) I added patient/Provenance.read to the MELD scope.  I committed this change to develop branch, and updated the MyCarePlanner scope in CarePlanning sandbox.

Provenance source is now displayed correctly in CarePlanning sandbox.  Pull and rebuild the app using changed I committed to develop branch to update MCC Development sandbox.

[mcc-provenance-pnoelle-hypertension.json.txt](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNWFkQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4b63ccb552af6076e7e247c04dfb11de36257306/mcc-provenance-pnoelle-hypertension.json.txt)
